lgc/CMakeLists.txt

```diff
@@ -95,8 +95,10 @@ if(WIN32)
 endif()
 
 target_include_directories(LLVMlgc
+PUBLIC
```

Comment:
I don't understand what this bit is doing - but based on your comments about making the interface public I guess this makes sense if it works when you tested it.